src/utils/ConfigLoader.ts

```diff
@@ -0,0 +1,20 @@
+import * as dotenv from 'dotenv';
+
+const DEFAULT_ENV_FILE = `${__dirname}/../../.env`;
+
+const loadConfigHelper = () => {
+  if (process.env.NODE_ENV) {
```

Comment:
It is still not quite right. Default config `.env` should be loaded at all times. Next environment specific config is loaded to override default / shared config if needed and add extra configuration. Something like:

```
dotenv.config();
if (process.env.NODE_ENV) {
  dotenv.config(`${__dirname}/../../.env.${process.env.NODE_ENV}`)
}
```

Reply:
I see